### ISSUE TITLE
Release builds of Mirth 3.6.0+ fails to start if java.security.debug property contains the "jar" flag/option

### ISSUE THREAD

**Issue by ab-20-m** [nextgenhealthcare/connect#4442]
If a user is trying to debug issues with a JCE or issue with JSSE, they will likely enabled the java.security.debug property. If the property is set to "jar" or "all", Mirth will fail to start with a StackOverflowException. This issues has been observed on both Windows and Linux platforms and using Oracle Java 8/11 and Oracle OpenJDK 8/11.

During startup with those options, as JARs are loaded by the JVM, each JAR is checked to see if it has been signed and if the signature is valid. Once the startup process gets to loading the serverlog-server.jar, something triggers a loop and a Stack Overflow. Disabling the Server Log extension in extensions.properties allows Mirth to start normally if java.security.debug is enabled. 

I don't think this is really a Mirth issue as much a JDK/JRE/JVM issue. However, I can only reproduce issue with a full-blown Mirth install. I've tried loading just the serverlog-*.jar in a "Hello World" app with java.security.debug=jar and the JAR verifies and loads without issue.

Some random thoughts on possible interactions/environment variables that might be triggering this:
1. The version of JDK used to sign the release JARs - Seems unlikely as loading just the serverlog-*.jars seems to work in a standalone app.
2. State of loaded JARs/classpath impacting something - Possible, but not clear what or how that would impact the JAR verification process
3. log4j redirection of stderr - The JAR verification process originally outputs to stderr before log4j loads and redirects and captures output. Shouldn't break anything but who knows.
4. log4j bug - Another possibility. The log messages start to just repeat concatenated output from the JAR verification process. 

Bottom line is the only people that could really debug this are JDK developers who also happen to know Mirth really well. The workaround is simple, just don't enable java.security.debug=jar/all. If you need to enable java.security.debug, disable the Server Log extension first.
